openwisp_users/tests/test_models.py

```diff
@@ -35,3 +35,10 @@ def test_organizations_pk_empty(self):
     def test_organization_repr(self):
         org = self._create_org(name='org1', is_active=False)
         self.assertIn('disabled', str(org))
+
+    def test_user_last_name(self):
+        long_string = ""
+        for i in range(151):
+            long_string += "a"
+        user = self._create_user(username=long_string)
+        self.assertEqual(len(user.username), 151)
```

Comment:
I don't understand why you're doing this, which lines are being tested by this?

Reply:
this [line](https://github.com/openwisp/openwisp-users/blob/master/openwisp_users/models.py#L66) I did that because I don't want to hard code string which consist of more than 150 chars